src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -271,6 +272,100 @@ public void parsesHttpBody() throws IOException {
     }
 
     /**
+     * RqMultipart.Base can close all parts once the request body has been
+     * closed.
+     * @throws IOException If some problem inside
+     */
+    @Test(expected = IOException.class)
```

Comment:
@essobedo even if first line of the test below throws `IOException`, the test would still pass. We need more precise assertion here.